gulpfile.js

```diff
@@ -142,9 +142,12 @@ gulp.task("sonarqube", callback => {
     });
   } else if (process.env.TRAVIS_PULL_REQUEST !== "false") {
     runSonnarQubeScanner(callback, {
+      "sonar.pullrequest.branch": process.env.TRAVIS_PULL_REQUEST_BRANCH,
+      "sonar.pullrequest.base": process.env.TRAVIS_BRANCH,
+      "sonar.pullrequest.key": process.env.TRAVIS_PULL_REQUEST,
+      "sonar.pullrequest.provider": "github",
+      "sonar.pullrequest.github.repository": process.env.TRAVIS_REPO_SLUG,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace double quotes by simple quote ([javascript:SingleQuote](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ASingleQuote))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=17&issues=AWMHRoj-aT4gCmfv8jYh&open=AWMHRoj-aT4gCmfv8jYh)

[ISSUE_KEY]: AWMHRoj-aT4gCmfv8jYh